Sprint-1/errors/3.js

```diff
@@ -1,5 +1,6 @@
-const cardNumber = 4533787178994213;
-const last4Digits = cardNumber.slice(-4);
+//const cardNumber = 4533787178994213;
+const cardNumber = "4533787178994213"; // slice() is a method for Array, and string is an Array of characters
```

Comment:
If you are not allowed to modify the original value of `cardNumber` (which is of a number type), how would you rewrite your code to get the last 4 digits?

Reply:
Thanks CJ for such interesting questions
I'll start with the fact that theory is not my strong point, but I'll try to give more precise answers
In the programming language I'm switching from now, I prefer these constructions:
```
const cardNumber = 4533787178994213;
const last4Digits = cardNumber.toString().slice(-4);
```
this guarantees that it will work for any type of `cardNumber`. But we can still come up with "scenarios" when this code won't work